build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.1'
+        classpath 'com.android.tools.build:gradle:4.1.0'
```

Comment:
Not sure if this needs to also go in the CHANGELOG?

Reply:
We could this may end up changing a good bit though. I'd put this in the same category as our more traditional gradle dependencies. I wonder if we should start logging dependency updates though overall?